README.md

```diff
@@ -298,6 +299,18 @@ store_options:
 
 These examples are by no way complete, moneta has much more to offer. Please have a look at [moneta's documentation](https://github.com/minad/moneta/blob/master/README.md) for further information.
 
+#### Vault backend
+
+For use vault you need to create un kv storage in your vault. The `store_options` are dynamic argument for initializer Vault::Client class (except `:kv`, used to defined the kv name)
+
+```YAML
+store: :vault
+store_options:
+  :kv: secret
```

Comment:
:kv could be named :kv_mount for less ambiguity, 'secret' is the default kv v1 mount name in vault, but it's nice to be able to choose the mount we want to use